### PR TITLE
Added syntax highlighting and copy buttons

### LINKED ISSUE #309: [Feature] Adding Copy button on top of codes for improved user experience
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

 For applying any code or feature its hard to copy all the code and might even result in copying the wrong code or leaving a bracket or two while selecting all. 

## **Describe the solution you'd like**

It would be really helpful if a "Copy" button is present that copies everything in just a single click. It solve a munch of hassles

For example:

image

which is not present in the current website 
image

### PR DESCRIPTION
**Pull request by NateNear**
# Description

Added syntax highlighting and copy buttons for a better user experience and less hassle

## Issue Ticket Number

Fixes #309

this fixes issue #309[Feature] Adding Copy button on top of codes for improved user experience
https://github.com/colbyfayock/next-cloudinary/issues/309

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Fix or improve the documentation
- [ ] This change requires a documentation update

# Checklist

- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have created an [issue](https://github.com/colbyfayock/next-cloudinary/issues) ticket for this PR
-  [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/colbyfayock/next-cloudinary/pulls) for the same update/change?
- [x] I have performed a self-review of my own code
- [x] I have run tests locally to ensure they all pass
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes needed to the documentation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,5 +9,7 @@
     "next-cloudinary",
     "scripts"
   ],
-  "devDependencies": {}
+  "dependencies": {
+    "react": "^18.2.0"
```
Comment (colbyfayock) [inline package.json:13]: was this an intentional add? dont think this is needed here 🤔 along with that i dont think the package-lock and yarn.lock changes are then related, can you revert those?

**Comment by NateNear** [comment]
@colbyfayock can you check again please

**Comment by NateNear** [comment]
Thanks a lot @colbyfayock 😃

**Comment by colbyfayock** [comment]
@all-contributors please add @NateNear for doc

**Comment by colbyfayock** [comment]
@NateNear congrats on the merged PR! since it's officially October, you've always qualified for Hacktoberfest swag! please email [hacktoberfest@cloudinary.com](mailto:hacktoberfest@cloudinary.com) with your github username and a link to your contribution where I'll follow up to request more information for getting you your swag

https://cloudinary.com/blog/hacktoberfest-celebrate-open-source-sdks

**Pull request merged**